src/schema/search.ts

```diff
@@ -274,62 +252,43 @@ export const resolvers: IResolvers<unknown, Context> = traceResolvers({
       getSession(ctx.userId, id),
     searchPostSuggestions: async (
       source,
-      { query, version }: { query: string; version: number },
+      { query }: { query: string; version: number },
```

Comment:
Don't use version here to avoid backward issues

Reply:
Additionally, what does this mean for local, search no longer works because we don't have meili locally?